src/game.rs

```diff
@@ -38,6 +41,15 @@ impl Deserialize for PlayerId {
     }
 }
 
+impl<'a> FromParam<'a> for PlayerId {
```

Comment:
This is used by Rocket's [request guard](https://rocket.rs/guide/requests/#request-guards) functionality. If the parameter is random junk and doesn't deserialize to a valid `PlayerId`, then the request won't match against the endpoint and Rocket will effectively return a 404 error (in our case it returns a 500, since we haven't fixed #15 yet).